src/higlass/widget.js

```diff
@@ -202,17 +203,38 @@ async function registerJupyterHiGlassDataFetcher(model) {
         });
         return response.payload;
       },
-      async fetchTiles({ tileIds }) {
-        let response = await sendCustomMessage(tModel, {
-          payload: { type: "tiles", tileIds },
-        });
-        let result = hgc.services.tileResponseToData(
-          response.payload,
-          NAME,
-          tileIds,
-        );
-        return result;
-      },
+      fetchTiles: consolidator(
+        /** @param {Array<WithResolvers<{ tileIds: Array<string> }, Record<string, any>>>} requests */
+        async (requests) => {
+          const batches = await Promise.all(
+            Array.from(
+              chunkIterable(
+                new Set(requests.flatMap((r) => r.data.tileIds)),
+                20,
```

Comment:
Where does this 20 come from? Arbitrary max chunk size?

Reply:
Yeah, I'll move to the top of the tile as a constant. I think it's 15 in HiGlass IIRC